common/djangoapps/course_modes/models.py

```diff
@@ -102,6 +102,7 @@ class Meta(object):
         max_length=255,
         null=True,
         blank=True,
+        default=None,  # Need this in order to set DEFAULT NULL on the database column
```

Comment:
Why does the empty value for this field need to be stored as NULL and not an empty string?